encrypted-media-respec.html

```diff
@@ -3825,25 +3825,25 @@
 
           <section id="per-origin-individualization">
             <h5>Per-Origin Individualization</h5>
-            <p>Per-<a def-id="origin"></a> individualization is performed between the <a def-id="cdm"></a> and an application-selected server and results in a per-origin identifier.
-              The process MUST be performed via these APIs and MUST NOT involve other types of communication.
-              Any <a def-id="distinctive-identifiers"></a> used MUST be origin-specific.
+            <p>Per-<a def-id="origin"></a> individualization is performed between the <a def-id="cdm"></a> and the application, including an application-selected server, and results in a per-origin identifier.
+              The process MUST be performed via these APIs and MUST NOT involve other methods of communication.
+              As with all other uses of the APIs, <a def-id="distinctive-identifiers"></a> MAY be used but indelible or non-origin-specific MUST NOT be exposed, even in encrypted form.
+              If one or more <a def-id="distinctive-identifiers"></a> is used, the resulting identifier is by definition also a <a def-id="distinctive-identifier"></a>.
```

Comment:
I believe this follows from the following bullet in the definition of Distinctive Identifier:
>The generation, individualization, provisioning or other process that produced the value involved, used, provided, derived from, or similarly involved such an indelible value or **another distinctive identifier.**

I'm open to considering scenarios like @mwatson2 suggests, but I think we should evaluate this separately. Shall we file a V1NonBlocking issue, reference it in an ISSUE box in the definition, and move forward on this CL, which is almost entirely unrelated to that issue?

@mwatson2, if you agree, please file the issue.